packages/utils/src/npm.ts

```diff
@@ -62,10 +70,16 @@ export const installPackages = async (packages: string[]): Promise<boolean> => {
             // Even if `hint` is installed locally, package.json could not exist in the current working directory.
             isDev = false;
         }
+
+        packageManagerChoice = (await hasYarnLock(currentWorkingDir)) ? 'yarn' : 'npm';
```

Comment:
Global installs should use `npm`, regardless of if the project itself uses `yarn` (also a good thing to add a test case for).
```suggestion
        packageManagerChoice = (!global && await hasYarnLock(currentWorkingDir)) ? 'yarn' : 'npm';
```

Reply:
@antross This statement is already in the `!global` block so `!global` check will be not required in the ternary operator.

Reply:
Ah, I missed that. Perhaps moving setting `packageManagerChoice` outside of the `if (!global)` block would make this clearer? Putting it immediately after the `const global=` statement would also allow making this a `const packageManagerChoice = ...` declaration instead of a `let`.

Reply:
Yeah, that would improve readability.